client/core/core.go

```diff
@@ -4899,10 +4939,16 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 		return nil, nil, newError(orderParamsErr, "zero-rate order not allowed")
 	}
 
-	wallets, err := c.walletSet(dc, form.Base, form.Quote, form.Sell)
+	wallets, assetConfigs, versionsMatch, err := c.walletSet(dc, form.Base, form.Quote, form.Sell)
 	if err != nil {
 		return nil, nil, err
 	}
+	// maybe we should just let the wallets decide in FundOrder if the server's
+	// version works? We'd have to confirm with the redeeming wallet too...
```

Comment:
Maybe change to a `SupportedVersions []uint32` field in `asset.WalletInfo`?

Reply:
Done.  Not entirely show how that's gonna jive with your contract version PR.